.github/workflows/fast-forward.yml

```diff
@@ -39,16 +38,7 @@ jobs:
 
       - name: Fast-forward merge
         run: |
+          echo "Fast-forward merging to ${{ inputs.target-ref }} branch from source ${{ inputs.source-ref }} branch"
```

Comment:
can we rephrase this to be "from A to B" instead of "to B from A"?

Reply:
done
```bash
echo "Fast-forwarding ${{ inputs.source-ref }} to ${{ inputs.target-ref }}"
```